books/management/commands/addepub.py

```diff
@@ -39,8 +39,6 @@ def validate_and_add(path, filenames):
         if os.path.splitext(path)[1] == '.epub':
             filename = os.path.abspath(path)
             if filename not in filenames:
-                if not models.Book.objects.filter(
-                        original_path=filename).exists():
```

Comment:
It seemed like the quick way for when running addepub against a folder of thousands of ePubs, to find a few new ones, to make it as quick as possible. I had forgotten about resync, though remembered it was there, and indeed I will use it someday!, and didn't check it at the time. The sha check happens late in the parsing, I believe after the cover is found and parsed, and I was hoping for a quicker way to run through a large set of files while not looking at dupes, or at least going through them as fast as possible. I'll try to review it sometime; if you have any ideas, they'd be most welcome!